graphql-dgs-spring-graphql/src/main/kotlin/com/netflix/graphql/dgs/apq/DgsAPQSupportAutoConfiguration.kt

```diff
@@ -93,7 +93,6 @@ open class DgsAPQSupportAutoConfiguration {
     )
     open class APQMicrometerMeteredCaffeineCacheConfiguration {
         @Bean
-        @ConditionalOnBean(io.micrometer.core.instrument.MeterRegistry::class)
```

Comment:
This conditional should still be there, no?

Reply:
Fixed.